src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -42,6 +42,7 @@ const useCoreChatSystemHook = () => {
   const {getAllSignedPostNotifications, getAllAnonymousPostNotifications} =
     useFetchPostNotificationHook();
   const {queue} = useDatabaseQueueHook();
+  const {updateAppBadgeFromDB} = useAppBadgeHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new hook `useAppBadgeHook` is introduced and its function `updateAppBadgeFromDB` is being used. Ensure that this hook is properly tested and handles potential errors to avoid unexpected behavior.
```javascript
const {updateAppBadgeFromDB} = useAppBadgeHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->